### PR TITLE
Feature/py3 full integration

### PR DESCRIPTION
**Pull request by nickumia-reisys**
This covers the remaining work to reintegrate all of the required extenstions in the Inventory.  There are quite a bit of changes to the extensions that needed to be made to pass integration tests.  The following PRs track those changes,
- https://github.com/GSA/ckanext-googleanalyticsbasic/pull/7
- https://github.com/GSA/ckanext-usmetadata/pull/55
- https://github.com/GSA/ckanext-datajson/pull/104
- https://github.com/GSA/ckanext-dcat_usmetadata/pull/270

### CODE REVIEW

File: config/development.ini [hunk 1]
```diff
@@ -135,8 +135,9 @@ solr_url = http://solr:8983/solr/inventory
 # Note: Add ``datastore`` to enable the CKAN DataStore
 #               Add ``resource_proxy`` to enable resource proxying and get around the
 #               same origin policy
-# TODO: re-integrate dcat_usmetadata, usmetadata, datajson, saml2auth
-ckan.plugins = datagov_inventory datastore xloader stats text_view recline_view googleanalyticsbasic s3filestore envvars
+# TODO: Figure out how to integrate `saml2auth` since there is no automated testing for
+# that extension as yet.
+ckan.plugins = datagov_inventory datastore xloader stats text_view recline_view googleanalyticsbasic s3filestore envvars dcat_usmetadata usmetadata datajson
```
Comment (jbrown-xentity) [inline config/development.ini:140]: So a weird thing that we should document here; envvars has to be at the end. It has to do with how the plugins are parsed, and making sure that the environment variables are injected properly before other extensions are run.
https://github.com/okfn/ckanext-envvars#requirements

File: e2e/cypress/integration/dcat_us_export.spec.js [hunk 2]
```diff
@@ -35,12 +35,12 @@ describe('DCAT-US Export', () => {
...
-    // it('Can create a zip export of the organization', () => {
-    //     cy.request('/organization/test-organization/draft.json')
-    //     .then((response) => {
-    //         cy.log(response.status, response.body);
-    //     })
-    // })
+    it('Can create a zip export of the organization', () => {
+        cy.request('/organization/test-organization/draft.json')
+        .then((response) => {
+            cy.log(response.status, response.body);
```
Comment (jbrown-xentity) [inline e2e/cypress/integration/dcat_us_export.spec.js:41]: We probably want to create expectations around this and not just log. Can we expect the response body to contain something?

File: env_file [hunk 3]
```diff
@@ -18,8 +18,7 @@ CKAN_HOME=/srv/app
 CKAN_CONFIG=/app/config
 PORT=5000
 
-# Need to add the following extensions: , dcat_usmetadata, usmetadata, datajson, saml2auth
-CKAN__PLUGINS=datagov_inventory datastore xloader stats text_view recline_view googleanalyticsbasic s3filestore envvars
+CKAN__PLUGINS=datagov_inventory datastore xloader stats text_view recline_view googleanalyticsbasic s3filestore envvars dcat_usmetadata usmetadata datajson saml2auth
```
Comment (jbrown-xentity) [inline env_file:21]: Same thing here, let's make sure envvars is at the end. Might need to disable saml2auth, not sure. We should make this less confusing by using the upstream config file, and only setting the configuration options here, but that's probably worth a separate ticket.

File: requirements.in.txt [hunk 4]
```diff
@@ -1,6 +1,7 @@
 git+https://github.com/ckan/ckan.git@ckan-2.9.3#egg=ckan
 -e git+https://github.com/GSA/ckanext-googleanalyticsbasic#egg=ckanext-googleanalyticsbasic
 -e git+https://github.com/GSA/ckanext-usmetadata.git@main#egg=ckanext-usmetadata
+-e git+https://github.com/GSA/ckanext-dcat_usmetadata.git@main#egg=ckanext-dcat_usmetadata
```
Comment (jbrown-xentity) [inline requirements.in.txt:4]: Are we going to revert this if we get on pypi? Should we wait until it's deployed?

**Review by jbrown-xentity — COMMENTED** [review]
Looking good, but some required changes before ready...

File: requirements.in.txt (hunk 4 above)
Comment (nickumia-reisys) [reply requirements.in.txt:4]: I have to update this.  Waiting to see if dcat_usmetadata deploys to pypi first though 😅

File: e2e/cypress/integration/dcat_us_export.spec.js (hunk 2 above)
Comment (nickumia-reisys) [reply e2e/cypress/integration/dcat_us_export.spec.js]: My test wasn't pushed when you reviewed it, but it's [here](https://github.com/GSA/inventory-app/pull/359/files#diff-ab4b110143d4b67e5a8a6f90bc6d4792448cb09ba8204dd74cc575feb365fec6R41-R53)

**Comment by nickumia-reisys** [comment]
For documentation purposes, this is the follow-on work for https://github.com/GSA/datagov-deploy/issues/3374.

File: requirements.in.txt (hunk 4 above)
Comment (jbrown-xentity) [reply requirements.in.txt:4]: It made it to pypi! https://pypi.org/project/ckanext-dcat-usmetadata/

Comment (nickumia-reisys) [reply requirements.in.txt:4]: Running tests locally with a new build and then will push.  After this is done, I don't think I know what to do with my life 😆

**Review by jbrown-xentity — APPROVED** [review]

**Pull request merged**